app/models/post.rb

```diff
@@ -0,0 +1,21 @@
+class Post < ApplicationRecord
+  belongs_to :user, optional: true
+  has_many :comments
+  has_many :likes
+
+  def delete_last
+    Post.last.destroy
+  end
+
+  def five_comments
+    Comment.first(5)
+  end
```

Comment:
- [ ] In the project description, we are asked to return the 5 most recent comments, but I think your comment returns the first five comments. I propose something like below
```ruby
def recent_comments
  comments.order('created_at Desc').limit(5)
end
```
- [ OPTIONAL] Also the method name is not the best. Consider now a situation of getting the 10 recent comments. A method name like recent_comments will be better